optimum/intel/openvino/configuration.py

```diff
@@ -464,14 +496,17 @@ def post_init(self):
                 raise ValueError(
                     f"When applying weight compression with '{self.weight_format}' weight format, the `bits` parameter must be set to 4, but found {self.bits}"
                 )
-            if self.quant_method == OVQuantizationMethod.AWQ:
-                raise ValueError(f"The AWQ algorithm is not supported for '{self.weight_format}' weight format")
-            if self.scale_estimation:
-                raise ValueError(
-                    f"The Scale Estimation algorithm is not supported for '{self.weight_format}' weight format"
-                )
-            if self.weight_format == "mxfp4" and self.gptq:
-                raise ValueError("The GPTQ algorithm is not supported for 'mxfp4' weight format")
+            if self.weight_format == "mxfp4":
+                if self.quant_method == OVQuantizationMethod.AWQ:
+                    raise ValueError("The AWQ algorithm is not supported for 'mxpf4' weight format")
+                if self.scale_estimation:
```

Comment:
Why not extend `OVQuantizationMethod`? Can also substitute `self.quant_method.name` to a single error message.

Reply:
> Why not extend OVQuantizationMethod?

Then we won't be able to apply AWQ + LoRA correction at the same time. I'm afraid, we shouldn't have relied on `OVQuantizationMethod` for enabling AWQ in the first place. But still stick to it now for backward-compatability.